marklogic-rdf4j/src/test/java/com/marklogic/semantics/rdf4j/MarkLogicRepositoryCacheTest.java

```diff
@@ -166,4 +167,34 @@ public void testLarge()
 //        conn.remove(stmts);
 //        assertEquals(0L, conn.size());
     }
+
+    @Test
+    public void testLargeTrig() throws Exception
+    {
```

Comment:
This test takes a long time to run.  Perhaps in the normal course of testing we should `@Ignore` it.
It would be good to understand, though, how long this takes in comparison to n-quads.  I suspect
that this trig loading is a lot slower.  If we can document our observations as we go, that's a good plus.